src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -62,10 +62,6 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
 
```

Comment:
@daxian-dbw, your last commit had 35 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Validate Update-Help from the Web for one PowerShell module..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for one PowerShell module for user scope..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'Microsoft.PowerShell.Host' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'Microsoft.PowerShell.Management' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```